x/concentrated-liquidity/grpc_query.go

```diff
@@ -133,6 +134,34 @@ func (q Querier) Params(goCtx context.Context, req *types.QueryParamsRequest) (*
 	return &types.QueryParamsResponse{Params: q.Keeper.GetParams(ctx)}, nil
 }
 
+// TickLiquidityInBatches returns array of liquidity depths in the given range of lower tick and upper tick.
+// Note that the space between the ticks in the returned array would always be guaranteed spacing greater than given batch unit.
+func (q Querier) TickLiquidityInBatches(goCtx context.Context, req *types.QueryTickLiquidityInBatchesRequest) (*types.QueryTickLiquidityInBatchesResponse, error) {
+	if req == nil {
```

Comment:
Is there a reason you don't have some utility or middleware to do basic request validation across all queries?.. it seems this is done on every query.